ansible/roles/installation/steamvr/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+# SteamVR installation
+- name: Update package lists
+  apt: update_cache=yes
+  become: yes
+
+- name: Intall needed libs for steamvr
+  apt: name={{ item }} install_recommends=no
+  become: yes
+  with_items:
+    - curl
+    - ca-certificates
+    - lib32gcc1
+    - binutils
+    - terminator
```

Comment:
you need terminator for SteamVR?

Reply:
Not strictly. It's used as the 'CMD' at the end of the dockerfile. Should I remove it?

Reply:
yes, please. Some dockerfiles will need it some not. This role is about steam VR installation.